zpm.c

```diff
@@ -19,6 +19,34 @@
 
 static char zpm_init[PATH_MAX];
 static char zpm_list[PATH_MAX];
+static int zpm_has_color = 0;
+static const char* zpm_usage = "Usage:\n\tzpm \"zsh-users/zsh-syntax-highlighting\"\n\tzpm disable \"zsh-users/zsh-syntax-highlighting\"\n\
+\tzpm remove \"zsh-users/zsh-syntax-highlighting\"\n\nAvailable commands:\n\tzpm reset\n\tzpm list\n\tzpm update\n\tzpm save\n\tzpm help\n\tzpm version";
+
+enum ZPM_MSG_TYPE {
+    ZPM_MSG,
+    ZPM_ERR,
+    ZPM_LOG,
+    ZPM_DEBUG,
+    ZPM_NONE
+};
+
+void zpm_msg(enum ZPM_MSG_TYPE type, const char* fmt, ...) {
+    char msg[PATH_MAX];
+    FILE* f = type == ZPM_ERR ? stderr : stdout;
+    va_list arg;
+    va_start(arg, fmt);
+    vsnprintf(msg, 256, fmt, arg);
+    if (type != ZPM_NONE) {
+       if(zpm_has_color) {
+        int color = type == ZPM_ERR ? 31 : 32;
```

Comment:
sorry, I mostly pasted untested code from the issue, so while fixing I forgot that 😕